### DISCUSSION TITLE
Change default behavior when clicking on photo

### DISCUSSION THREAD

**Discussion by CWempe** [photoprism/photoprism#2060] [Ideas]
Currently when clicking on a photo it gets selected.
I have to click the zoom icon in the left top corner to open a photo.

Coming from Google Photos I prefer it the other way round:

* Clicking anywhere on the photo should open the photo viewer.
* Clicking the circle in the right bottom corner selects the photo.

If you want to select multiple photos fast, you could hold the `Ctrl` key to select a photo without targeting the little symbol.

I guess you wont change the default behavior just for me. :)
But would it be possible to let the user define this in the settings?

**Comment by graciousgrey** [comment]
If you click on a photo anywhere except the circle it does open in the photo viewer.

In case you select one photo you are in selection mode and clicking on further photos selects them. This is by design.
If you want to select photos in fullscreen mode you can select them there and then use the arrows to view the next/previous photos.

Do I get you right that you want an option to disable selection mode? This could be done eventually but given our current [workload](https://github.com/photoprism/photoprism/issues) and [funding](https://github.com/sponsors/photoprism?preview=true) it will take a while as there are many tasks with higher priority :)

**Reply by kvalev** [reply]
Not on mobile - there you have to click the magnifying glass in the top left corner to open the image (both in cards and mosaic view).

**Reply by graciousgrey** [reply]
On mobile you also do not need to click on the magnifying glass. If you do a normal/short click on an image  it opens in fullscreen mode. 

If you do a long click on an image it get's selected and selection mode is on. Further normal clicks will select additional photos. Further long clicks will select multiple images at once (similar to shift +click on desktop).

**Reply by CWempe** [reply]
> If you click on a photo anywhere except the circle it does open in the photo viewer.

When I click in the middle of a photo, it gets selected, not opened.

![photoprism_click_photo2](https://user-images.githubusercontent.com/1270359/155429994-6907e85b-cb79-4c70-bc10-e17c8564bb58.gif)

Windows 10
Chrome Browser

**Reply by kvalev** [reply]
If you take a look at the bottom right, there are already 13 selected photos. When you clear the selection it will behave like you expect it to.

**Comment by CWempe** [comment]
You are right. 👍 
Shame on me.

But I did not expect the selection to last after closing and reopening the web site.

**Reply by lastzero** [reply]
Shall we remove this feature?

**Comment by CWempe** [comment]
It is kind of unintuitive, I think.

But can live with it.
I leave the decision up to you or other users.

**Reply by Alestrix** [reply]
First day user here (20k photos indexed so far and counting 😄). I also stumbled across this and was surprised the selection survived when leaving and re-entering a folder. But once one is aware of this behavior I think it's a feature that will prove practical once one starts some organizing tasks.

So I'd vote against removal

**Reply by graciousgrey** [reply]
We have many users using this feature ! So we rather won't remove it. We might add a config option later to disable it optionally.

**Comment by tsvetkovv** [comment]
To bring this a bit up, the issue (?) with a Live-photos. I didn't find the way open Live-photo as a photo when I am in a selection mode. If you press the left-top circle, the video is shown up. 
![image](https://user-images.githubusercontent.com/11594423/183580097-4ad9e929-a569-4cb9-aac3-d68d8a1fb673.png)
